src/models/user.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import Column, Integer, String, DateTime
+from sqlalchemy.orm import relationship
+
+from src.db.db import Base
+
+
+class User(Base):
+    # classes will inherit from Base
+    __tablename__ = "users"
+    # set tablename for use in db
+
+    id = Column(Integer, primary_key=True, index=True)
+    first_name = Column(String(50))
+    last_name = Column(String(50))
+    email = Column(String(100), unique=True, index=True)
+    phone_number = Column(String)
```

Comment:
String length?

Reply:
Could you clarify the question / comment here?

Reply:
The `phone_number` is a string but the length is not specified.

Reply:
Added!